### PR TITLE
Vanilla bars with category text

### PR DESCRIPTION
**Pull request by gracetian6**
**Added left labels for each corresponding right bar.** Data is still hardcoded in JS.

Achieved this by 
- creating multiple div block containers for each label. This text container should align with the corresponding bar.
- vertically centering text within its container

image

image

### CODE REVIEW

File: client/css/bar.css [hunk 1]
```diff
@@ -35,4 +35,16 @@
...
+  vertical-align: top;
+}
+
+.category {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  float: right;
+
+  text-align: right;
```
Comment (laptou) [inline client/css/bar.css:49]: Why do you have `float: right`?

Comment (seunomonije) [reply client/css/bar.css:49]: ^

File: client/js/bar.js [hunk 2]
```diff
@@ -4,13 +4,27 @@ const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
...
 document.getElementById('graph').appendChild(svg);
 
+// textBlock should align vertically with bar chart
+const barTextContainer = document.getElementById('bar-text');
+barTextContainer.setAttribute('height', svg.getBoundingClientRect().height);
+
 // hard coded data for bar chart for now
 const counts = [1, 3, 1, 2];
 const maxCount = 3;
+const genres = [
```
Comment (geekster777) [inline client/js/bar.js:14]: nit: this should be in SCREAMING_SNAKE_CASE

File: client/js/bar.js [hunk 3]
```diff
@@ -4,13 +4,27 @@ const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
 svg.setAttribute('viewBox', '0 0 100 100'); // change height in css
 document.getElementById('graph').appendChild(svg);
 
+// textBlock should align vertically with bar chart
+const barTextContainer = document.getElementById('bar-text');
+barTextContainer.setAttribute('height', svg.getBoundingClientRect().height);
```
Comment (geekster777) [inline client/js/bar.js:9]: This should go after the consts.

Comment (geekster777) [inline client/js/bar.js:9]: This is a bit fragile. You should be able to solve this with css. Try something like `height: 100%` and let me know if you need help debugging.

Comment (gracetian6) [reply client/js/bar.js]: TODO on tuesday

**Pull request merged**